schedview/collect/visits.py

```diff
@@ -143,7 +143,9 @@ def read_ddf_visits(*args, **kwargs) -> pd.DataFrame:
     ddf_field_names = tuple(ddf_locations().keys())
     ddf_visits = []
     for ddf_name in ddf_field_names:
-        this_field_visits = all_visits.query(f"{target_column_name}.str.contains('{ddf_name}')")
+        this_field_visits = all_visits.query(
+            f"{target_column_name}.str.contains('{ddf_name}') or scheduler_note.str.contains('DD:{ddf_name}')"
+        )
```

Comment:
Will this catch our variable capitalization? 
I think we've tried to make target_name values match the observation_reason values (which are enforced lower-case), but I didn't mess with the scheduler_note values. 
It's a pain.